src/Features/LanguageServer/Protocol/Microsoft.CodeAnalysis.LanguageServer.Protocol.csproj

```diff
@@ -24,7 +24,6 @@
   </ItemGroup>
 
   <ItemGroup>
-    <InternalsVisibleTo Include="Microsoft.CodeAnalysis.ExternalAccess.LiveShare" />
```

Comment:
So this basically is the new external access assembly for live share?

Reply:
Kind of.  This contains all the stuff needed for liveshare integration.  But Liveshare does not have IVTs to this or any other assembly anymore.